src/components/typography/typography.spec.tsx

```diff
@@ -627,6 +679,37 @@ describe("Typography", () => {
   });
 
   testStyledSystemSpacing((props) => <Typography {...props} />);
+
+  it("applies the tabIndex when tabIndex prop is supplied", () => {
+    const wrapper = mount(
+      <ThemeProvider theme={mintTheme}>
+        <Typography tabIndex={-1}>FooBar</Typography>
+      </ThemeProvider>
+    );
+
+    expect(wrapper.find(Typography).prop("tabIndex")).toEqual(-1);
+  });
+
+  it("applies the ref when ref prop is supplied", async () => {
+    const typographyHandle = React.createRef<TypographyHandle>();
+
+    const MockComponent = () => {
+      return (
+        <ThemeProvider theme={mintTheme}>
+          <Typography ref={typographyHandle} tabIndex={-1} data-role="heading">
+            FooBar
+          </Typography>
+        </ThemeProvider>
+      );
+    };
+    render(<MockComponent />);
+
+    expect(typeof typographyHandle.current?.focus).toBe("function");
+
+    await typographyHandle.current?.focus();
```

Comment:
**suggestion(non-blocking)**: `focus()` doesn't return a promise so we don't necessarily need the `await` here.

Reply:
Outdated/removed now